ATTOCUBE/iocBoot/iocATTOCUBE-IOC-01/st-common.lua

```diff
@@ -0,0 +1,38 @@
+device = "LO"
+motor_port = "MTR"
+num_axes = 3
+pv_prefix = os.getenv("MYPVPREFIX")
+p = string.format("%s%s", pv_prefix, os.getenv("IOCNAME"))
+recsim = os.getenv("RECSIM") == "1"
+devsim = os.getenv("DEVSIM") == "1"
+emulator_port = os.getenv("EMULATOR_PORT")
+if(emulator_port==nil) then
+    emulator_port=57677
+end
+
+if(recsim) then
+    iocsh.drvAsynSerialPortConfigure(device, "NUL", 0, 1, 0, 0)
+else
+    if(devsim) then
+        connection_string = string.format("localhost:%i", emulator_port)
+    else
+        connection_string = string.format("%s:2101", os.getenv("IP_ADDR"))
+    end
+	print("Connecting on " .. connection_string)
+    iocsh.drvAsynIPPortConfigure(device, connection_string)
+    iocsh.anc350AsynMotorCreate(device, "0", "0", num_axes)
+    iocsh.drvAsynMotorConfigure(motor_port, "anc350AsynMotor", "0", num_axes)
+end
+
+iocsh.dbLoadRecords("db/common.db", string.format("P=%s,PORT=%s", p, device))
+
+for axis_num=1,num_axes,1
+do
+    motor_pv = string.format("MTR%02i%02i", os.getenv("MTRCTRL"), axis_num)
+    single_axis_db = "db/single_axis.db"
+    db_args = string.format("P=%s,MYPVPREFIX=%s,MOTOR_PV=%s,CONTROLLER_PORT=%s,MOTOR_PORT=%s,ADDR=%s", p, pv_prefix, motor_pv, device, motor_port, axis_num-1)
+    iocsh.dbLoadRecords(single_axis_db, db_args)
+end
+
+#iocsh.asynSetTraceIOMask(device, -1, 0x2)
+#iocsh.asynSetTraceMask(device, -1, 0x9)
```

Comment:
I get the following error when running the IOC tests:

```
#iocsh.asynSetTraceIOMask(device, -1, 0x2)
lua: stdin:1: attempt to get length of a nil value
stack traceback:
	stdin:1: in main chunk
#iocsh.asynSetTraceMask(device, -1, 0x9)
lua: stdin:1: attempt to get length of a nil value
stack traceback:
	stdin:1: in main chunk
```

This seems to be because `#` is the `length` operator in lua - I suspect you meant for this to be a comment?

It also means it's currently spamming the log with every command sent and received, which would very quickly fill up an instrument disk.